packages/@uppy/aws-s3/src/index.js

```diff
@@ -124,6 +124,9 @@ module.exports = class AwsS3 extends BasePlugin {
 
     this.opts = { ...defaultOptions, ...opts }
 
+    // TODO: remove i18n once we can depend on XHRUpload instead of MiniXHRUpload
```

Comment:
Why do we want to switch from `XHRUpload` to `MiniXHRUpload`?

Reply:
Not sure, it's based on this comment: https://github.com/transloadit/uppy/blob/0f9709bb101ca2fc9f690fc79611b97fde209cf7/packages/@uppy/aws-s3/src/index.js#L272

Reply:
@goto-bus-stop could you elaborate?

Reply:
See this:
https://github.com/transloadit/uppy/blob/5a96886103fc214df425e2c449aa1a82aed319c9/packages/%40uppy/aws-s3/src/index.js#L1-L26

MiniXHRUpload is a workaround. After a successful pipeline refactor, the `@uppy/xhr-upload` plugin can be used by the S3 plugin for the actual uploading. So we want to eventually get rid of MiniXHRUpload, and then we can also get rid of that line.